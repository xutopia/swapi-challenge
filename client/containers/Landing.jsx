import React, { Component } from 'react';
import { connect } from 'react-redux';

export class Landing extends Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <div>
        Hello Landing
      </div>
    )
  }
}

export default connect()(Landing);
