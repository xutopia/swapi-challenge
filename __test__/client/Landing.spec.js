import renderer from 'react-test-renderer'
import { Landing } from '../../client/containers/Landing.jsx';

describe('Landing --- Snapshot',() => {
  it('+++capturing Snapshot of Landing', () => {
    const renderedValue =  renderer.create(<Landing />).toJSON()
    expect(renderedValue).toMatchSnapshot();
  });
});
