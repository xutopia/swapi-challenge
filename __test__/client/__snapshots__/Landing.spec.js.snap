// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Landing --- Snapshot +++capturing Snapshot of Landing 1`] = `
<div>
  Hello Landing
</div>
`;
